htdocs/mrp/class/api_mos.class.php

```diff
@@ -368,297 +381,177 @@ public function produceAndConsume($id, $request_data = null)
 		}
 
 		if (empty($labelmovement)) {
+			$error++;
```

Comment:
This line should be useless because there is a throw just after, so we live the function